<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>internetDialog</class>
 <widget class="QDialog" name="internetDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>565</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string> network settings</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>190</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#f7f0f0;&quot;&gt;adaptive strategy:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>190</y>
     <width>201</width>
     <height>27</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Buffer-Based Adaptation</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Buffer-Based AdaptationLogicAggressive</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Buffer-Based AdaptationLogicNormal</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>No AdaptationLogic</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>AlwaysLowest AdaptationLogic</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Rate-Based AdaptaptionLogic</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>400</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>cancel</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_3">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>260</y>
     <width>201</width>
     <height>27</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>25</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>50</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>100</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>120</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#fcf2f2;&quot;&gt;forwarding strategy:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>260</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#f5e5e5;&quot;&gt;cache size(M):&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:18pt; color:#e5bbbb;&quot;&gt;network settings&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>400</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>ok</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>120</y>
     <width>201</width>
     <height>27</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>BestRoute Strategy</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>NCC Strategy</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Multicast Strategy</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Client-control Strategy</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>330</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#fdf6f6;&quot;&gt;topology:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_4">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>330</y>
     <width>201</width>
     <height>27</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>10 nodes topology</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>50 nodes topology</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
